src/main/java/homework_7/Main.java

```diff
@@ -0,0 +1,16 @@
+package homework_7;
+
+import homework_7.kitten_to_cat_function.Cat;
+import homework_7.kitten_to_cat_function.Kitten;
+import homework_7.kitten_to_cat_function.KittenToCatFunction;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Kitten kitten = new Kitten("Murzik", 1);
+        KittenToCatFunction function = (k) -> new Cat(k.getName(), k.getAge() + 4);
```

Comment:
Opt: you could add some more complicated logic to lambda, as example - converting one type to another (like, favourite toy to favourite color or smth like this)